backend/src/main/java/ch/nova_omnia/lernello/model/data/LearningKit.java

```diff
@@ -53,12 +53,16 @@ public class LearningKit {
     @Column(name = "context")
     private String context;
 
+    @Column(name = "createdAt", nullable = false)
+    @NotNull
+    private ZonedDateTime createdAt = ZonedDateTime.now();
```

Comment:
I thought there is an `@` annotation for setting the default date, but if it works, it works